### PR TITLE
feat(Google Play): add presence

### LINKED ISSUE #5488: Google Play | play.google.com
### Discussed in https://github.com/PreMiD/Presences/discussions/4491

<div type='discussions-op-text'>

<sup>Originally posted by **Isa27-0** September 16, 2020</sup>
**Prerequisites and essential questions**
- [x] Is it a popular site?
    °The website is from the Android app store
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?**

**What's the name of the service?** 
Google Play (or Google Play Store): https://play.google.com

**What should the Presence display?** Service name, product (movie, book, music or app) author's name and time on page

**If possible, please provide a logo for the service (512x512 minimum)** 
google-play-logo-3 (1)

</div>

### PR DESCRIPTION
**Pull request by AngeloCore**
**Describe the changes in this pull request:**

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

</details>

Resolves #5488

### CODE REVIEW

File: websites/G/Google Play/presence.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+              "div.D0ZKYe > div.rlnrKc > div.sIskre > c-wiz > h1.AHFaub > span"
+            ).textContent;
+
+          if (logo) presenceData.largeImageKey = await getShortURL(logo);
+
+          presenceData.details = `Viewing ${name ? name : "app"}`;
+          presenceData.buttons = [
+            {
+              label: "View app",
+              url: await getShortURL(document.location.href)
```
Comment (Slowlife01) [inline websites/G/Google Play/presence.ts:95]: The max length here is 512 not 256. I doubt it'd get that long?

Comment (AngeloCore) [reply websites/G/Google Play/presence.ts:95]: I really don't know if the url length of the image will be more than 512, but the `getShorURL` method returns the original url if it's not more than 512, I know that will make the presence slower.
Should I remove it?

Comment (Slowlife01) [reply websites/G/Google Play/presence.ts:95]: I meant buttons' url not imageKey

Comment (AngeloCore) [reply websites/G/Google Play/presence.ts:95]: The default app url should be `https://play.google.com/store/apps/details?id=com.instagram.android` for example

But if Google adds some queries, and sometimes they put a lot of queries, the URL will get more than 512 characters (ig)

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**